src/handlers/push_message.rs

```diff
@@ -175,18 +178,52 @@ pub async fn handler_internal(
         )
     })?;
 
+    let cloned_body = body.clone();
+    let push_message = if client.always_raw {
```

Comment:
Probably this condition or our spec description should be changed because in our spec we [require the tag to be presented](https://github.com/WalletConnect/walletconnect-specs/pull/147/files#diff-25490e5dda4bcd46802fca4131439d45ab85cf408683b76c7604735dfe656edcR82) as well if always_raw == false to filter by tag and send clear text notification for some of the tags. I think we should fix this in a spec and do tag filtering when the always_raw == true. cc @chris13524

Reply:
> When set to `false`, cleartext notifications will be sent for certain messages based on `tag`

This filtering is [performed in the relay](https://github.com/WalletConnect/rs-relay/pull/1285/files#diff-7644ae4ab3ab50067abfcd3582ce60dcf0eade53bc256dea0acf373f4bdbfb57R554), and was already implemented. So I don't think anything needs to be done here.